tests/test_core.py

```diff
@@ -377,6 +378,8 @@ def test_500_io(self):
         aman.wrap('sparse', csr_array( ((8,3), ([0,1], [1,54])), 
                                       shape=(aman.dets.count, aman.samps.count)))
 
+        aman.wrap('quantity', np.ones(5) << u.m)
```

Comment:
Looking good but something is up with scalars ... If I add this:
```
        aman.wrap('quantity2', (np.ones(5) << u.m)[0])
```
it causes the test to fail.

Reply:
Looks like it is because scalar Quantities are size-0 arrays which seems to  mess with some stuff that h5py does when it is loading it back. I can add a separate schema for scalar quants to get around this.